src/codemagic/tools/_app_store_connect/action_groups/builds_action_group.py

```diff
@@ -223,6 +305,9 @@ def submit_to_app_store(
             support_url,
             whats_new,
         )
+        if cancel_previous_submissions:
+            self._cancel_previous_submissions(build_id=build_id, platform=platform)
```

Comment:
This could be called as part of `_submit_to_app_store`. Then additional request for build's application won't be required.